src/components/Wallet/WalletGenerator.jsx

```diff
@@ -49,6 +47,8 @@ import {
 } from "../../actions/clientActions";
 import { MAX_FETCH_UTXOS_ERRORS, MAX_TRAILING_EMPTY_NODES } from "./constants";
 
+// const bitcoin = require('bitcoinjs-lib');
```

Comment:
cleanup.

Reply:
leaving these. it's how you fix a problem with testing and can be cleaned up in the future